### PR TITLE
Update WR (details below):

### PR DESCRIPTION
**Pull request by glennw**
* Add support for clip masks on text runs.
* Fix atomic ordering of items with multiple shadows.
* Update to bincode + ipc-channel with optimizations.
* Fix some plane splitting precision errors.
* Improve the anti-aliasing quality significantly.
* Add internal ClipChain support.
* Fix diacritic glyphs on Linux.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/18890)

### CODE REVIEW

**Comment by glennw** [comment]
There are two new failures in this WR update.

They are caused by https://github.com/servo/webrender/issues/1776, which is being exposed by the vast improvements to anti-aliasing landed in https://github.com/servo/webrender/pull/1822.

r? @jdm or @emilio

File: tests/wpt/metadata-css/compositing-1_dev/html/mix-blend-mode-intermediate-element-overflow-hidden-and-border-radius.htm.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[mix-blend-mode-intermediate-element-overflow-hidden-and-border-radius.htm]
+  type: reftest
```
Comment (emilio) [inline tests/wpt/metadata-css/compositing-1_dev/html/mix-blend-mode-intermediate-element-overflow-hidden-and-border-radius.htm.ini]: Can you add a `bug: <link-to-issue>` entry here?

File: tests/wpt/metadata-css/css-transforms-1_dev/html/transform-3d-rotateY-stair-below-001.htm.ini [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[transform-3d-rotateY-stair-below-001.htm]
+  type: reftest
```
Comment (emilio) [inline tests/wpt/metadata-css/css-transforms-1_dev/html/transform-3d-rotateY-stair-below-001.htm.ini]: Same here.

**Review by emilio — APPROVED** [review]

**Comment by glennw** [comment]
Sure, done.

**Pull request merged**